src/main/java/core/basesyntax/service/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.service.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(File file) {
+        try {
+            return Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
```

Comment:
The exception message is not informative enough. It should include the file path to provide more context about the error.